apps/qaqc/src/cpdb/helpers.py

```diff
@@ -48,22 +44,21 @@ def get_data(branch, previous_branch, previous_version) -> dict:
         "geometries": ["cpdb_dcpattributes_pts", "cpdb_dcpattributes_poly"],
     }
 
-    client = DigitalOceanClient(bucket_name=BUCKET_NAME, repo_name=REPO_NAME)
-
     for t in tables["analysis"]:
-        rv[t] = client.csv_from_DO(url=construct_url(branch, t, sub_folder="analysis/"))
-        rv["pre_" + t] = client.csv_from_DO(
-            url=construct_url(
-                previous_branch, t, previous_version, sub_folder="analysis/"
-            )
+        rv[t] = publishing.read_csv(
+            DATASET, f"{branch}/latest/output", f"analysis/{t}.csv"
```

Comment:
maybe for clarity with how this function is being used now, `version` arg should be called `output` since really it makes most sense to be pointing to output folder.